src/utils/client.js

```diff
@@ -175,26 +200,59 @@ export default class Client {
     return this._fetch(url, options).then((response) => response.json());
   }
 
+  async fetchForm(initialValues) {
+    const result = await this._fetchCacheForm();
+    const steps = getABVariant(result);
+    this.setDefaultFormValues({ steps, additionalValues: initialValues });
+    return [steps, result];
+  }
+
   async fetchSession() {
-    const { userKey, sessions, authId } = initInfo();
+    const {
+      userKey,
+      sessions,
+      authId,
+      fieldValuesInitialized: noData
+    } = initInfo();
     if (this.formKey in sessions)
       return Promise.resolve(sessions[this.formKey]);
 
-    const params = encodeGetParams({
-      form_key: this.formKey,
-      ...(userKey ? { fuser_key: userKey } : {}),
-      ...(authId ? { auth_id: authId } : {})
-    });
+    initState.fieldValuesInitialized = true;
```

Comment:
should this function still proceed if `fieldValuesInitialized` is already true?

Reply:
yes it should because there is other session data to initialize

Reply:
for some reason I wasn't appreciating that `fieldValuesInitialized: noData` is a rename - I think I processed it as a default value. I think I'm now following how you are using this 👍